ci-operator/config/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master.yaml

```diff
@@ -96,21 +118,43 @@ tests:
   steps:
     cluster_profile: aws
     test:
+    - as: install
+      cli: latest
+      commands: |
+        oc create ns openshift-vertical-pod-autoscaler
+        operator-sdk run bundle --timeout=10m -n openshift-vertical-pod-autoscaler --security-context-config restricted "$OO_BUNDLE"
+        oc wait --for condition=Available -n openshift-vertical-pod-autoscaler deployment/vertical-pod-autoscaler-operator
+      dependencies:
+      - env: OO_BUNDLE
+        name: vertical-pod-autoscaler-operator-bundle
+      from: operator-sdk
+      resources:
+        requests:
+          cpu: 100m
     - as: test
       cli: latest
       commands: make e2e-olm-ci
-      dependencies:
-      - env: CI_OPERATOR_IMAGE
-        name: vertical-pod-autoscaler-operator
-      - env: CI_OPERAND_IMAGE
-        name: vertical-pod-autoscaler
-      - env: CI_REGISTRY_IMAGE
-        name: vpa-operator-registry
       from: src
       resources:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: scorecard
+  steps:
+    allow_best_effort_post_steps: true
+    cluster_profile: aws-2
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:multilatest
+    test:
+    - as: test
+      cli: latest
+      commands: make test-scorecard
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: hypershift-hostedcluster-workflow
```

Comment:
This is clever, running the scorecard tests against a hypershift node because it's less expensive. Was this cribbed off another operator or is this bespoke?

Reply:
Noticed an example in the docs :D There is an example here: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-a-cluster-from-hypershift

Reply:
heck yeah, love it, clearly I should go check the docs again periodically :smile: